src/libmongoc/doc/mongoc_client_new_with_error.rst

```diff
@@ -0,0 +1,35 @@
+:man_page: mongoc_client_new_with_error
+
+mongoc_client_new_with_error()
+==============================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  mongoc_client_t *
+  mongoc_client_new_with_error (const char *uri_string, bson_error_t *error)
```

Comment:
I vote to not including `mongoc_client_new_with_error`.

IMO it does not provide much convenience over `mongoc_client_new_from_uri_with_error`.

`mongoc_client_pool_new` accepts a `mongoc_uri_t`. There are no requests I know of to add a variant that accepts a string.

Reply:
Removed.